presto-docs/src/main/sphinx/installation/installation-ambari.rst

```diff
@@ -0,0 +1,111 @@
+
+.. code-block:: bash
+
+	$ tar -xvf /path/to/integration/package/ambari-presto-0.1.0.tar.gz -C /var/lib/ambari-server/resources/stacks/HDP/2.3/services/PRESTO
+	$ mv /var/lib/ambari-server/resources/stacks/HDP/2.3/services/PRESTO/ambari-presto-0.1.0/* /var/lib/ambari-server/resources/stacks/HDP/2.3/services/PRESTO
+	$ rm -rf /var/lib/ambari-server/resources/stacks/HDP/2.3/services/PRESTO/ambari-presto-0.1.0
+
+* Finally, make all integration files executable and restart the Ambari server:
+
+.. code-block:: bash
+
+	$ chmod -R +x /var/lib/ambari-server/resources/stacks/HDP/2.3/services/PRESTO/*
+	$ ambari-server restart
+
+* Once the server has restarted, point your browser to it and on the main Ambari Web UI page click the ``Add Service`` button and follow the on screen wizard to add Presto. The following sections provide more details on the options and choices you will make when adding Presto.
+
+Supported Topologies
+--------------------
+The following two screens will allow you to assign the Presto processes among the nodes in your cluster. Once you pick a topology for Presto and finish the installation process it is impossible to modify that topology.
+
+Presto is composed of a coordinator and worker processes. The same code runs all nodes because the same Presto server RPM is installed for both workers and coordinator. It is the configuration on each node that determines how a particular node will behave. Presto can run in pseudo-distributed mode, where a single Presto process on one node acts as both coordinator and worker, or in distributed mode, where the Presto coordinator runs on one node and the Presto workers run on other nodes.
+
+The client component of Presto is the ``presto-cli`` executable JAR. You should place it on all nodes where you expect to access the Presto server via this command line utility. The ``presto-cli`` executable JAR does not need to be co-located with either a worker or a coordinator, it can be installed on its own. Once installed, the CLI can be found at ``/usr/lib/presto/bin/presto-cli``.
+
+**Do not place a worker on the same node as a coordinator**. Such an attempt will fail the installation because the integration software will attempt to install the RPM twice. In order to schedule work on the Presto coordinator, effectively turning the process into a dual worker/coordinator, please enable the ``node-scheduler.include-coordinator`` toggle available in the configuration screen.
+
+Pseudo-distributed
+^^^^^^^^^^^^^^^^^^
+Pick a node for the Presto coordinator and **do not assign any Presto workers**. On the configuration screen that follows, you must also enable ``node-scheduler.include-coordinator`` by clicking the toggle.
+
+Distributed
+^^^^^^^^^^^
+
+Pick a node for the Presto coordinator and assign as many Presto workers to nodes as you'd like. Feel free to also place the client component on any node. Remember to not place a worker on the same node as a coordinator.
```

Comment:
Replace "assign as many Presto workers to nodes as you'd like" with "assign Presto workers to as many nodes as you'd like"

If only one coordinator is allowed, replace "a coordinator" with "the coordinator".